docs/MigrationGuide.md

```diff
@@ -27,6 +27,13 @@ To see the biggest differences please consult the [changelog](Changelog.md).
   which provide the same interface as the previous one, with only method name changed.
   Especially, when you use 'PeriodicDeploymentManagerProvider', `delegate` should already return `DeploymentManager` wrapped by caching mechanism.
 * [#4131](https://github.com/TouK/nussknacker/pull/4131) `Parameter.defaultValue` now holds `Option[Expression]` instead of `Option[String]`. You have to wrap a `String` with `Expression.spel()`
+* [#4132](https://github.com/TouK/nussknacker/pull/4132) 
```

Comment:
WDYT about somehow merge'ing this entry with #4122? So developer will see all changes in statuses in one place. Or at least move it close to it?